### PR TITLE
npm/bump

### LINKED ISSUE #158: Update NPM packages
## Description
- Update outdated npm packages.

### PR DESCRIPTION
**Pull request by csimonovsky**
# Description
Updated outdated npm packages.

### Test Coverage
- [x] unit

#### References
- Closes #158

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1,12232 +1,8 @@
 {
   "name": "@briza/illogical",
   "version": "1.4.1",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```
Comment (stevenzhao95) [inline package-lock.json:4]: Should this be changed? iirc if the local version of npm is out of date this will be changed

@davidhorak for more insight

Comment (davidhorak) [reply package-lock.json:4]: @csimonovsky do you have latest node?

**Comment by davidhorak** [comment]
@csimonovsky in the PR description you have the unit test unchecked, did you checked it?

File: package-lock.json (hunk 1 above)
Comment (csimonovsky) [reply package-lock.json:4]: I needed to update npm, but im using node v15.3.0

Comment (csimonovsky) [reply package-lock.json]: c1e5fca

**Comment by stevenzhao95** [comment]
Not sure if it's normal for that large amount of deletions in `package-lock.json`

@csimonovsky was that from running `npm update`?

**Comment by csimonovsky** [comment]
> Not sure if it's normal for that large amount of deletions in `package-lock.json`
> 
> @csimonovsky was that from running `npm update`?

Yeah I just followed the steps in https://docs.google.com/document/d/1CrxCbQbcEF6wAA4q2r09Jxb-SaNtgCAqupRRXntLbvA/edit#heading=h.ovtu8a9hut2q

**Comment by davidhorak** [comment]
> > Not sure if it's normal for that large amount of deletions in `package-lock.json`
> > @csimonovsky was that from running `npm update`?
> 
> Yeah I just followed the steps in https://docs.google.com/document/d/1CrxCbQbcEF6wAA4q2r09Jxb-SaNtgCAqupRRXntLbvA/edit#heading=h.ovtu8a9hut2q

before the npm update or after?
It may be worth while to get back before npm update commit and rerun the npm update.

**Review by stevenzhao95 — APPROVED** [review]
LGTM. @davidhorak do you mind taking another look to see if I missed anything?

**Review by davidhorak — APPROVED** [review]
Tests are passing, if build-able (build-js), than LGTM

**Pull request merged**